src/Components/Web.JS/src/Rendering/SymbolUtil.ts

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+export function createSymbolOrFallback(fallback: string): symbol | string {
+  return typeof Symbol === 'function' ? Symbol() : fallback;
+}
```

Comment:
I think we're OK to just require `Symbol` to be supported now: https://caniuse.com/?search=symbol